main.go

```diff
@@ -244,13 +230,6 @@ func run(conf configuration, listen string, debug bool) {
 		router.PathPrefix(prefix).Handler(http.StripPrefix(prefix, http.FileServer(http.Dir(parsedURL.Path))))
 	}
 
-	go func() {
-		time.Sleep(5 * time.Minute)
-		ag.stats.Incr(foobarTestCounterName, []string{"statsd:yes"}, 1)
-		foobarTestCounter.Inc()
-		promOnlyFoobarTestCounterName.Inc()
-	}()
-
```

Comment:
(note) This might be a hack we might end up using when sending metrics about config.